CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Xcodeproj Changelog
 
+## Master
+
+##### Enhancements
+
+* Add method to create new variant groups (groups for localized versions of the same file)  
+  [Tim Bodeit](https://github.com/timbodeit)
+  [Xcodeproj#301](https://github.com/CocoaPods/Xcodeproj/pull/315)
```

Comment:
Very good catch! I think I copied the line to match the rest of the changelog and only changed one of the PR numbers.